### PR TITLE
Query Youtube API using api key only

### PR DESCRIPTION
**Pull request by amCap1712**
Found that this works by hit and try and some chance. All documentation points to need of Oauth to access the Youtube but somehow this works 😄 . We tested this on beta. This also renders most of the Youtube Oauth stuff we wrote useless 😭 , will remove that in a subsequent PR.

### CODE REVIEW

File: listenbrainz/webserver/templates/user/music_services.html [hunk 1]
```diff
@@ -72,32 +72,9 @@ <h3 class="panel-title">Youtube</h3>
             <div class="panel-body">
                 <p>
```
Comment (mayhem) [inline listenbrainz/webserver/templates/user/music_services.html:73]: I wonder if we should allow people to disable playing from YT as a preference.

**Review by mayhem — APPROVED** [review]
:crossed_fingers: :crossed_fingers: :crossed_fingers: :crossed_fingers: :crossed_fingers: :crossed_fingers: :crossed_fingers: :crossed_fingers: :crossed_fingers: :crossed_fingers:

File: listenbrainz/webserver/templates/user/music_services.html (hunk 1 above)
Comment (amCap1712) [reply listenbrainz/webserver/templates/user/music_services.html:73]: @MonkeyDo had some plans about implementing preferences in BrainzPlayer. Maybe revisit when we do that?

Comment (mayhem) [reply listenbrainz/webserver/templates/user/music_services.html:73]: Yeah, perfect.

**Pull request merged**